src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -426,8 +426,34 @@ typedef struct MPIDI_OFI_win_request {
     struct fi_context context[MPIDI_OFI_CONTEXT_STRUCTS];       /* fixed field, do not move */
     int event_id;               /* fixed field, do not move */
     struct MPIDI_OFI_win_request *next;
+    struct MPIDI_OFI_win_request *prev;
     int target_rank;
     int rma_type;
+    union {
+        struct {
+            /* origin */
+            const void *origin_addr;
```

Comment:
`MPI_Get` cannot use `const` for the origin.

Reply:
Ah, yes. Will fix.

Reply:
For now I've reduced scope to only MPI_PUT and left this as `const void *`.